integrations/utils/currency/currency.js

```diff
@@ -8,6 +9,17 @@ export function convertToEuro(amount, currency) {
   return amount / exchangeRate;
 }
 
+export function convertTo2011Euro(amount, currency) {
```

Comment:
The `utils/currency/currency` file is shared between many systems. To keep things isolated, I think it's best if the JSON file is added to the purchase/ folder, and the methods be added to purchase/index.js instead of currency.js.

Reply:
I think I've done what was needed here.